app/views/main_layout.scala.html

```diff
@@ -85,6 +85,11 @@
         message = Some(messages("timeout.dialog.message.text"))
         )
     )
+@appConfig.gtmContainer.map { containerId =>
```

Comment:
minor, but can you indent new code to match code around it

Reply:
Done